universal-application-tool-0.0.1/test/app/ApplicantBrowserTest.java

```diff
@@ -0,0 +1,19 @@
+package app;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class ApplicantBrowserTest extends BaseBrowserTest {
+
+  @Test
+  public void applicantProgramList_selectApply_redirectsToEdit() {
+    addProgram("My Program");
+    goTo(controllers.applicant.routes.ApplicantProgramsController.index(1L));
```

Comment:
suggestion **(if-minor)**: logout as admin and login as applicant/guest before visiting the applicant view

Reply:
Good idea - I added a logout method and changed addProgram to login/logout as admin. The login as admin is a bit tricky, since I want to try returning the applicant ID in the method - but working on it!